packages/schemix/src/schemix/database.py

```
@@ -0,0 +1,42 @@
+
+CType = TypeVar("CType", bound=Mapping[str, ColumnType | SQLExpression])
+
+
+class Database:
+    """Main database interface for schemix.
+
+    Provides query building capabilities and schema management for SQLite and PostgreSQL.
+    Supports type-safe queries through mypy plugin integration.
+
+    Args:
+        connection: Pre-configured AsyncConnection instance (SQLiteConnection or PostgreSQLConnection)
```

Pylint (3.3.7)

[convention] Line too long (102/100)

(C0301)